contracts/bancor-portal/BancorPortal.sol

```diff
@@ -349,22 +359,47 @@ contract BancorPortal is IBancorPortal, ReentrancyGuardUpgradeable, Utils, Upgra
         uint256 poolTokenAmount
     ) private {
         IERC20(address(pair)).safeApprove(address(router), poolTokenAmount);
-
         uint256 deadline = block.timestamp + MAX_DEADLINE;
-        if (tokens[0].isNative()) {
-            router.removeLiquidityETH(address(tokens[1]), poolTokenAmount, 1, 1, address(this), deadline);
-        } else if (tokens[1].isNative()) {
-            router.removeLiquidityETH(address(tokens[0]), poolTokenAmount, 1, 1, address(this), deadline);
+
+        Token token0 = _nativeToWETH(tokens[0]);
+        Token token1 = _nativeToWETH(tokens[1]);
```

Comment:
Can get rid of these two, and use `if (tokens[x].isNative())` below, where `x` is `0` or `1`.

Reply:
done